src/generics/package.lisp

```diff
@@ -12,10 +12,11 @@ You can view their documentation in their respective API sections.
 
 The main documentation for the functionality is given here, with
 examples often given in the specific methods"
-  (gapply     pax:generic-function)
-  (maybe      pax:generic-function)
-  (to-circuit pax:generic-function)
-  (to-bitc    pax:generic-function)
-  (to-poly    pax:generic-function)
-  (to-cat     pax:generic-function)
-  (to-vampir  pax:generic-function))
+  (gapply         pax:generic-function)
+  (maybe          pax:generic-function)
+  (to-circuit     pax:generic-function)
+  (to-bitc        pax:generic-function)
+  (to-seqn        pax:generic-function)
+  (to-poly        pax:generic-function)
+  (to-cat         pax:generic-function)
+  (to-vampir      pax:generic-function))
```

Comment:
we should maybe move this to its own commit or if this is due to a new addition to that commit.